From ef76eb4d944a1a0a7ce98caa488560431655e2bd Mon Sep 17 00:00:00 2001
From: Kendall Willis <k-willis@ti.com>
Date: Mon, 15 Jun 2026 10:40:07 -0500
Subject: [PATCH 2/3] perf(ti): Enable the auto self refresh configurations for
 DDR

For low power HMI use case, enable auto self refresh for DDR.

Change-Id: If48de5406ac163a2f57b91890d1abe92cedcdfb2
Signed-off-by: Kendall Willis <k-willis@ti.com>
---
 fdts/am62lx-skevm-lp4-50-800.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/fdts/am62lx-skevm-lp4-50-800.dtsi b/fdts/am62lx-skevm-lp4-50-800.dtsi
index 9daba8d6b..a65cde57a 100644
--- a/fdts/am62lx-skevm-lp4-50-800.dtsi
+++ b/fdts/am62lx-skevm-lp4-50-800.dtsi
@@ -183,9 +183,9 @@
 #define DDRSS_CTL_164_DATA 0x08080801
 #define DDRSS_CTL_165_DATA 0x0000080E
 #define DDRSS_CTL_166_DATA 0x00040003
-#define DDRSS_CTL_167_DATA 0x00000007
-#define DDRSS_CTL_168_DATA 0x00000000
-#define DDRSS_CTL_169_DATA 0x00000000
+#define DDRSS_CTL_167_DATA 0x07074007
+#define DDRSS_CTL_168_DATA 0x00000f07
+#define DDRSS_CTL_169_DATA 0x0303000f
 #define DDRSS_CTL_170_DATA 0x00000000
 #define DDRSS_CTL_171_DATA 0x00000000
 #define DDRSS_CTL_172_DATA 0x00000000
-- 
2.34.1

